packaging/fedora/snapd.spec

```diff
@@ -426,6 +426,8 @@ install -m 644 -D data/info %{buildroot}%{_libexecdir}/snapd/info
 
 # Install bash completion for "snap"
 install -m 644 -D data/completion/snap %{buildroot}%{_datadir}/bash-completion/completions/snap
+install -m 644 -D data/completion/complete.sh %{buildroot}%{_libexecdir}/snapd
+install -m 644 -D data/completion/etelpmoc.sh %{buildroot}%{_libexecdir}/snapd
```

Comment:
Thanks for reviewing
It is where other scripts are looking for to make snap-completion. 
Any suggestion about how to improve it?